de_workloads/ingest/ingest_azure_sql_example/data_factory/adf_linked_services.tf

```diff
@@ -1,11 +1,11 @@
-resource "azurerm_data_factory_linked_custom_service" "ls_AzureSql_ExampleSource" {
-  name            = var.linked_service_name
+resource "azurerm_data_factory_linked_custom_service" "ls_azure_sql_example" {
+  name            = "ls_azure_sql_example"
```

Comment:
The approach using variables for these sorted of values would mean setting the linked service name we require as the default value for the variable (as we're not passing things like that at runtime). We'd just be setting that value in the jinja template in the variables, rather than where it's actually getting used. 
We decided this approach was preferable as it made the generated terraform a bit easier to understand.

Reply:
if we're using a value multiple times we can use a `local`

Reply:
We could look at doing that - hesitant to adjust the templates too much as part of this PR though.